query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/CswSpatialOperator.java

```diff
@@ -78,14 +78,14 @@ public String getValue() {
 
   public static final class Bbox implements EnumValue<String> {
 
-    public static final String BBOX = "BBOX";
+    public static final String BBOX_ENUM = "BBOX";
```

Comment:
Are these colliding with the class name? I'd prefer we rename the class to `BboxEnumValue` instead. That constant is used both as the enum value and enum title so renaming it `BBOX_ENUM` doesn't add much value to the variable name. Most of the values changed in the other enums look to be the same case.

Reply:
Yes, they are. If you prefer to rename the classes I'll go ahead and do that.

Reply:
We still need to change `EQUALS` because it clashs with the `java.lang.Object`'s equals.
https://github.com/connexta/admin-console/pull/165/files/7954ff878b54791d3f62c4b933e9259080b328a1#diff-598a6b0dbf9a7c21b0231c1949182983L219

Reply:
I'm gonna go ahead and change the enum strings to what they were and change the class name to `XxxEnumValue` format